book-1-orientation/chapters/STUDENT_EXERCISES_TYPES.md

```diff
@@ -0,0 +1,84 @@
+# Tracking Student Exercises: Custom Types
+
+You are going to build a console application that tracks exercises that are assigned to students at Nashville Software School. These are the constraints and requirements for your application.
+
+## Entity Relationship Diagram
```

Comment:
Move this section beneath the sections where you itemize the properties for each class. Let them read the descriptions before asking them to make an ERD